requirements.txt

```diff
@@ -0,0 +1,9 @@
+Cerberus==1.3.4
+certifi==2021.5.30
+elasticsearch==7.13.3
```

Comment:
@jmazanec15 may be 7.10.2?

Reply:
Do we really need dependency to elasticsearch library here?

Reply:
Yeah 7.10.2 makes sense. We need it in order to communicate with the OpenSearch cluster. I think there may be an OpenSearch python client we can use, but I am not sure it is ready: https://discuss.opendistrocommunity.dev/t/opensearch-client-java-python-apis/5639/20. Until then, the elasticsearch python client should work and it has AL2 license.

Reply:
Why not use request library ?